packages/unity-interface/initializer.ts

```diff
@@ -101,7 +101,6 @@ async function loadInjectedUnityDelegate(
   const transport = webTransport({ wasmModule: originalUnity.Module }, (globalThis as any).DCL)
 
   await engineStartedFuture
-  await browserInterface.startedFuture
```

Comment:
this was preventing the RPC from starting...

this is already being awaited here: https://github.com/decentraland/kernel/blob/e9c174b4e2ae20fba7309a3703778c434da41b65/packages/unity-interface/dcl.ts#L48

Reply:
Does it still work in desktop? I remember this was a bugfix